src/main/webapp/WEB-INF/logging.properties

```diff
@@ -0,0 +1,13 @@
+# A default java.util.logging configuration.
+# (All App Engine logging is through java.util.logging by default).
+#
+# To use this configuration, copy it into your application's WEB-INF
+# folder and add the following to your appengine-web.xml:
+# 
+# <system-properties>
+#   <property name="java.util.logging.config.file" value="WEB-INF/logging.properties"/>
+# </system-properties>
+#
+
+# Set the default logging level for all loggers to WARNING
+.level = INFO
```

Comment:
comment doesn't match setting.

Reply:
Nice. Suggested comment: "Set the default logging level for all loggers." to avoid the mismatch altogether.